Samples/3.DebugLog/Program.cs

```diff
@@ -76,6 +76,8 @@ static void Main( string[] args )
 
             // create our steamclient instance
             steamClient = new SteamClient();
+            steamClient.DebugNetworkListener = new NetHookNetworkListener();
```

Comment:
I'd probably make it a separate sample with warnings everywhere, or leave it unsampled. It seems fairly simple enough to add into any pre-existing sample.